stac_api/runtime/src/templates/stac-viewer.html

```diff
@@ -662,7 +662,7 @@
 
 map.on('load', () => {
   // get collections
-  fetch('{{ endpoint }}/collections')
+  fetch('{{ endpoint }}/api/stac/collections')
```

Comment:
this is configurable so we should try to pass the config.py `root_path` here if possible (there may be an example of doing this with the template for the cog viewer in the raster api). If not we should capture the need in our back log so we are not surprised in the future if veda-backend deployed with a different root path

Reply:
https://github.com/NASA-IMPACT/veda-backend/pull/311/commits/bdd898cc4f55e984a6971184f934b60c7edf9374